suite-native/receive/src/utils.ts

```diff
@@ -0,0 +1,50 @@
+import { A, pipe, S } from '@mobily/ts-belt';
+import { RequireAllOrNone } from 'type-fest';
+
+import { splitAddressToChunks } from '@suite-native/helpers';
+import { DeviceModelInternal } from '@trezor/connect';
+
+import { DevicePaginationActivePage } from './types';
+
+const filterAddressChunksByPagination = (
+    addressChunks: readonly string[],
+    activePage: DevicePaginationActivePage,
+) => {
+    const paginationEmptyChunk = '    ';
+    if (activePage === 1) {
+        return [...addressChunks.slice(0, 15), paginationEmptyChunk];
+    }
+
+    return [paginationEmptyChunk, ...A.sliceToEnd(addressChunks, 15)];
+};
+
+export const parseAddressToDeviceLines = ({
```

Comment:
Can please get touch with @AdamSchinzel and sync with him how this is going to be done in desktop? I guess this parsing is something that can be reused, so we can avoid some duplicities.

Reply:
@AdamSchinzel I've checked your implementation and i can see you have a similar logic in your [DeviceDisplay component ](https://github.com/trezor/trezor-suite/blob/2189bb6d2b314b4045099a54018e548173f4581e/packages/suite/src/components/suite/DeviceDisplay.tsx). If I would move this method to `suite-common`, would you use it? Or do you see any complication in that? 

you would have to refactor a bit though, because you are working with JSX directly in your "chunking function".